src/Core/Commander/Providers/IoDriverXML.js

```diff
@@ -19,31 +19,9 @@ define('Core/Commander/Providers/IoDriverXML', ['Core/Commander/Providers/IoDriv
     IoDriverXML.prototype.constructor = IoDriverXML;
 
     IoDriverXML.prototype.read = function(url) {
-
-
-        return new Promise(function(resolve, reject) {
-            var xhr = new XMLHttpRequest();
-
-            xhr.open("GET", url, true);
-
-            xhr.responseType = "document";
-
-            xhr.crossOrigin = '';
-
-            xhr.onload = function() {
-                resolve(this.response);
-
-            };
-
-            xhr.onerror = function() {
-
-                reject(Error("Error IoDriverXML"));
-
-            };
-
-            xhr.send(null);
+        return fetch(url).then(function(response) {
+            return response.text();
```

Comment:
Oh, just happens to me that there's no direct `fetch` equivalent to `responseType="document"`, except of course parsing the XML explicitly with `DOMParser` **and** sniffing the encoding and turning the binary response to a string.

Assuming the XML is always in UTF-8, that'd read:

```js
var parser = new DOMParser();
return parser.parseFromString(response.text(), "application/xml");
```

(this is not strictly equivalent, besides the encoding issue, as the XHR would have parsed the response as HTML if the server had returned an XML content-type, and would have returned an `SVGDocument` if the response had an SVG content-type; but I think we can make a few assumptions and take a few shortcuts here)

Either that or keep using `XMLHttpRequest` here, and add a comment as to why it doesn't use `fetch`.